sources/packages/web/src/assets/css/base.scss

```diff
@@ -142,3 +142,21 @@ ul.no-bullets {
 .p-tabview-panels {
   margin-top: 16px;
 }
+
+.header-title {  
+  font-size: $font-header-medium;
+  color: $brand-black-light;
+  line-height: $line-height-header-md;
+  font-weight: $formio-header-wt;
```

Comment:
I believe that this style should be moved into the _base.scss_ and then consumed into form.io, not the opposite.

Reply:
It is currently in base.scss.

Reply:
The code is using the $formio-header-wt style from formio scss, I believe, right?
What I was saying is that the font-weight should be defined at base.scss and then consume on formio.